### PR TITLE
Add plugin test code document

### PR DESCRIPTION
**Pull request by cosmo0920**

### CODE REVIEW

**Comment by cosmo0920** [comment]
@fujimotos Any thoughts?

File: docs/v1.0/plugin-test-code.txt [hunk 1]
```diff
@@ -1,32 +1,835 @@
...
+
+    :::ruby
+    d.run do
+      d.feed("test", event_time, { "message" => "Hello, Fluentd!!" })
+    end
+
+
+#### feed(tag, array_event_stream)
+
+Feed an array of event stream to plugin instance.
```
[the remark this answers is not in the document]
Comment (cosmo0920) [reply docs/v1.0/plugin-test-code.txt]: `#feed` handles one array event stream.
`Fluent::OneEventStream` or `Fluent::MultiEventStream`'s instance can put into array_event_stream.
Both classes inherits `Fluent::EventStream` which is called "event stream" here.

**Comment by cosmo0920** [comment]
I've fixed them.
Do these fixes make sense?

File: docs/v1.0/plugin-test-code.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (fujimotos) [reply docs/v1.0/plugin-test-code.txt]: OK. If you have some rationale behind it, then I think you can go with it.

**Comment by fujimotos** [comment]
Thanks you. I've merged it.

**Comment by cosmo0920** [comment]
Thanks!

**Pull request merged**